modules/vaos/app/models/vaos/appointment_request_form.rb

```diff
@@ -0,0 +1,155 @@
+
+module VAOS
+  class AppointmentRequestForm < Common::Form
+    attribute :email, String
+    attribute :phone_number, String
+    attribute :option_date1, String # TODO: ideally these would be better abstracted as an array for a FE consumer
+    attribute :option_time1, String
+    attribute :option_date2, String
+    attribute :option_time2, String
+    attribute :option_date3, String
+    attribute :option_time3, String
+    attribute :status, String
+    attribute :appointment_type, String
+    attribute :visit_type, String
+    attribute :text_messaging_allowed, Boolean
+    attribute :phone_number, String
+    attribute :purpose_of_visit, String
+    attribute :other_purpose_of_visit, String
+    attribute :purpose_of_visit, String
+    attribute :provider_id, String
+    attribute :second_request, Boolean
+    attribute :second_request_submitted, Boolean
+    attribute :provider_name, String
+    attribute :requested_phone_call, Boolean
+    attribute :type_of_care_id, Boolean
+    attribute :has_veteran_new_message, Boolean
+    attribute :has_provider_new_message, Boolean
+    attribute :provider_seen_appointment_request, Boolean
+    attribute :requested_phone_call, Boolean
+    attribute :type_of_care_id, String
+    attribute :patient_id, String # This setter is overriden
+    attribute :appointment_request_id, String # This setter is overriden
+    attribute :unique_id, String # This setter is overriden
+    attribute :id, String # This setter is overriden
+    attribute :date, String # Automatically set for create if id is present; This setter is overridden
+    attribute :created_date, VAOS::AppointmentTime # Only passed in when updating
+    attribute :last_access_date, VAOS::AppointmentTime # Only passed in when updating
+    attribute :last_updated_date, VAOS::AppointmentTime # Only passed in when updating
+    attribute :assigning_authority, String, default: 'ICN'
+    attribute :system_id, String, default: 'var'
+    attribute :object_type, String, default: 'VARAppointmentRequest'
+    attribute :surrogate_identifier, Hash, default: {}
+    attribute :facility, Hash
+    attribute :patient, Hash, default: { inpatient: false, text_messaging_allowed: false } # This setter is overriden
+    attribute :best_timeto_call, Array[String] # VAR camel cases this incorrectly so we will snakecase it incorrectly
+    attribute :appointment_request_detail_code, Array[String] # This setter is overriden
+
+    def initialize(user, json_hash)
+      @user = user
+      @id = json_hash[:id]
+      @appointment_request_id = @id # ensure these are the same (and the FE doesn't have to pass in redundant data)
+      @unique_id = @id # ensure these are the same (and the FE doesn't have to pass in redundant data)
+      @date = Time.current.strftime('%Y-%m-%dT%H:%M:%S.%L%z') if @id.blank? # only set for create in proper format
+      @patient_id = edipi # TODO: discover if the patient_id ever comes from elsewhere or if it is always EDIPI
```

Comment:
This `TODO` should either be addressed & removed or extracted to an issue & removed from the code

Reply:
That's actually very sound advice. I'll go ahead and remove it and add a ticket to follow up with VAOS SMEs on this.